lib/model/Product.dart

```diff
@@ -42,8 +42,32 @@ class Product extends JsonObject {
   OpenFoodFactsLanguage lang;
   @JsonKey(includeIfNull: false)
   String quantity;
+
+  // Images
+  @Deprecated('imgSmallUrl is deprecated, use imageFrontSmallUrl instead')
```

Comment:
Shouldn't it be `@deprecated` instead of `@Deprecated`?

Reply:
if I put @deprecated('imgSmallUrl is deprecated, use imageFrontSmallUrl instead') instead of @Deprecated, I get an error: "Annotation must be either a const variable reference or const constructor invocation"

if I just put @deprecated (without a message), there's no error, but I don't see a warning either.

Reply:
My personal use is a comment (`///`) and then `@deprecated`. Then the deprecated method.
I've never used `@Deprecated`.
Perhaps `@JsonKey` then `@Deprecated`?...